ghcide/ghcide.cabal

```diff
@@ -60,7 +60,7 @@ library
         haddock-library >= 1.8 && < 1.11,
         hashable,
         hie-compat ^>= 0.2.0.0,
-        hls-plugin-api ^>= 1.2.0.2,
+        hls-plugin-api ^>= 1.2.0.2 || ^>= 1.3,
```

Comment:
Any reason not to drop the old constraints in the places where you've done this?

Reply:
No really, i changed all packages to use the same hls-graph, hls-plugin-api and ghcide bounds in all of them and make updates easier. The bounds includes the current version and the previous one but i did not any test if the prev version works or does not (or if the package works with older versions)

Reply:
AFAICT, the only thing we can make promises about is "the current state of the repo works", so I think including the previous versions isn't reliable (and does it benefit anyone?).

Reply:
agree will keep only the lastest one